content/posts/sharing-data-via-torrent.md

```diff
@@ -44,31 +44,33 @@ To understand how this may impact a network, we first need to discuss how torren
 >
 >That's fine. You can enjoy your book and help others get pages too.
 
-This comes with some advantages: first, with a torrent, once the first few people start downloading, they also become seeders, meaning later downloaders can get pieces of the file from multiple sources instead of relying solely on your network. This has potential to vastly reduce the outgoing bandwidth over direct downloading, especially if you multiple people are trying to download the data simultaneously.
+This comes with some advantages: 
 
-Next, it can increase the reliability of downloads. If the data server goes offline, you can still collect file parts from seeders and peers, whereas with a direct download, a server crash may cause the download to fail. Speaking of not relying on one server, another advantage of decentralization is sharing the load among all users participating in the torrent. This can reduce costs and help reduce the effects felt by server limitations.
+First, with a torrent, once the first few people start downloading, they also become seeders, meaning later downloaders can get pieces of the file from multiple sources instead of relying solely on your network. This has potential to vastly reduce the outgoing bandwidth over direct downloading, especially if multiple people are trying to download the data simultaneously.
 
-Finally, torrenting has great scalability. If we have a bunch of people trying to direct download from the data server at the same time this can put immense stress on the network slowing the downloads to a crawl. However, when torrenting, each downloader is sharing bandwidth and reducing the stress on any one server.
+Second, it can increase the reliability of downloads. If the data server goes offline, you can still collect file parts from seeders and peers, whereas with a direct download, a server crash may causing the download to fail. Speaking of not relying on one server, another advantage of decentralization is sharing the load among all users participating in the torrent. This can reduce costs and help reduce the effects felt by server limitations.
+
+Finally, torrenting scales exceptionally well. If a large number of users try to download data from a central server at the same time, this can put immense load on a network, causing performance degradation. However, when downloading data via BitTorrent, each client is sharing bandwidth, reducing the load on any one server.
```

Comment:
Improved sentence flow and prefer network terminology like "load" instead of "stress' or strain"